### PR TITLE
Placeholder text

### PR DESCRIPTION
**Pull request by bradgreen3**
-Add placeholder text to sign up fields
-Add stackoverflow to user and sign up 
-Add stackoverflow acceptance testing

### CODE REVIEW

File: spec/acceptance/user/completes_max_registration_spec.rb [hunk 1]
```diff
@@ -17,6 +17,8 @@
     fill_in 'user[linked_in]', with: user[:linked_in]
     fill_in 'user[git_hub]', with: user[:git_hub]
     fill_in 'user[slack]', with: user[:slack]
+    fill_in 'user[stackoverflow]', with: user[:stackoverflow]
```
Comment (AliSchlereth) [inline spec/acceptance/user/completes_max_registration_spec.rb]: Yay test coverage!

File: app/serializers/single_user_serializer.rb [hunk 2]
```diff
@@ -1,5 +1,17 @@
 class SingleUserSerializer < ActiveModel::Serializer
-  attributes :id, :first_name, :last_name, :cohort, :image_url, :email, :slack, :twitter, :linked_in, :git_hub, :groups, :roles
+  attributes :id,
```
Comment (AliSchlereth) [inline app/serializers/single_user_serializer.rb]: I love how organized this is 😃

**Comment by AliSchlereth** [comment]
Looks ready to merge.

File: spec/acceptance/user/completes_max_registration_spec.rb (hunk 1 above)
Comment (bradgreen3) [inline spec/acceptance/user/completes_max_registration_spec.rb]: And lack of underscores!

**Pull request merged**